content/en/network_monitoring/devices/netflow.md

```diff
@@ -204,10 +204,53 @@ To visualize the real data sent by your devices, you can query for **Bytes (Samp
 
 ## Retention
 
-NetFlow data is retained for 30 days by default, with options for 15, 30, 60, and 90 day retention. 
+NetFlow data is retained for 30 days by default, with options for 15, 30, 60, and 90 day retention.
 
 <div class="alert alert-danger">To retain NetFlow data for longer periods of time, contact your account representative.</div>
 
+## Troubleshooting
+
+### Netflow Packet Drops
+Netflow packet drops can occur when there is a high number of Netflow packets per second, typically greater than 50,000. Here are some steps to identify and mitigate Netflow packet drops:
+
+#### Identifying Packet Drops
+
+You can use the netstat -s command to see if there are any dropped UDP packets:
+```bash
```

Comment:
For these code samples you'll want to indent 4 spaces to the right so they land underneath the mitigation steps, I can help with that if you need!